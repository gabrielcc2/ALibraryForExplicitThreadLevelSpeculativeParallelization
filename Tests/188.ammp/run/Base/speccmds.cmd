-u /home/gabriel/Escritorio/SPEC_CPU2000_CD_2/benchspec/CFP2000/188.ammp/run/Base
-i ammp.in >ammp.out 2>ammp.err ../Base/ammp_parallel
