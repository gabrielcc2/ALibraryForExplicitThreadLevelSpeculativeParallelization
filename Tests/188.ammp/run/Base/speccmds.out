running commands in speccmds.cmd 1 times
runs started at 1311548306, 489616000, Sun Jul 24 18:28:26 2011
run 1 started at 1311548306, 489696000, Sun Jul 24 18:28:26 2011
child started: 0, 1311548306, 489708000, 2691, '>ammp.out 2>ammp.err ../00000006/ammp_base.x86_linux'
child finished: 0, 1311548653, 147525000, 346, 657817000, 2691, 0
run 1 finished at: 1311548653, 147536000, Sun Jul 24 18:34:13 2011
run 1 elapsed time: 346, 657840000, 346.657840000
runs finished at 1311548653, 147582000, Sun Jul 24 18:34:13 2011
runs elapsed time: 346, 657966000, 346.657966000
